app/(auth)/forgot-password.tsx

```
@@ -0,0 +1,271 @@
+      // Request a password reset with Clerk
+      await signIn.create({
+        strategy: 'reset_password_email_code',
+        identifier: email
+      });
+
+      setResetSent(true);
+      toast.showToast('Password reset instructions sent', 'success');
+    } catch (error: any) {
+      console.error('Password reset error:', error);
+      
+      if (error.errors && error.errors[0]) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)